6.X/6.14.1/README.md

```diff
@@ -58,7 +58,7 @@ This install assumes the Kubernetes cluster has network access to pull images fr
     provided with your Sysdig purchase confirmation mail
   - [`sysdig.platformAuditTrail.enabled`](docs/02-configuration_parameters.md#sysdigplatformAuditTrailenabled):
     Set this parameter to `true` if you would like to use Sysdig Platform Audit.
-  - [`sysdig.secure.events.audit.config.store.ip.enabled`](docs/02-configuration_parameters.md#sysdigsecureeventsauditconfigstoreipenabled):
+  - [`sysdig.secure.events.audit.config.store.ip`](docs/02-configuration_parameters.md#sysdigsecureeventsauditconfigstoreipenabled):
```

Comment:
@mvalin pls fix the anchor:
`sysdigsecureeventsauditconfigstoreipenabled` should be `sysdigsecureeventsauditconfigstoreip`

Reply:
@mvalin I think you're missing the second file to fix.